packages/gatsby-plugin-sharp/src/gatsby-node.js

```diff
@@ -215,4 +215,26 @@ exports.pluginOptionsSchema = ({ Joi }) =>
     stripMetadata: Joi.boolean().default(true),
     defaultQuality: Joi.number().default(50),
     failOnError: Joi.boolean().default(true),
+    defaults: Joi.object({
+      formats: Joi.array().items(
+        Joi.string().valid(`auto`, `png`, `jpg`, `webp`, `avif`)
+      ),
+      placeholder: Joi.string().valid(
+        `tracedSVG`,
+        `dominantColor`,
+        `blurred`,
+        `none`
+      ),
+      quality: Joi.number(),
+      breakpoints: Joi.array().items(Joi.number()),
+      backgroundColor: Joi.string(),
+      tracedSVGOptions: Joi.object(),
+      blurredOptions: Joi.object(),
+      jpgOptions: Joi.object(),
+      pngOptions: Joi.object(),
+      webpOptions: Joi.object(),
+      avifOptions: Joi.object(),
+    }).description(
```

Comment:
transformOptions? Any of the sizing stuff like aspectRatio?

Reply:
I don't think the sizing options make sense to have as global defaults. I guess transformOptions could though. I'll add them.

Reply:
Ya, my one thought was aspectRatio could, since it's more look and feel that strict sizing. But no strong feelings on that one.